python/cudnn/gemm/cutedsl/dense/swiglu/api.py

```
@@ -120,45 +144,44 @@ def check_support(self) -> bool:
+                    dtype=[
+                        cutlass.Float32,
+                        cutlass.Float16,
+                        cutlass.BFloat16,
+                        cutlass.Float8E4M3FN,
+                        cutlass.Float8E5M2,
+                    ],
+                    name="AB12 (for float32 acc_dtype)",
+                )
+                self._not_implemented_error_if(
+                    self._is_fp8(self.ab12_dtype),
+                    f"ab12_dtype {{torch.float8_e5m2, torch.float8_e4m3fn}} is currently disabled",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)